### ISSUE TITLE
huggingface

### ISSUE THREAD

**Issue by vaclavmuller** [chrisdonahue/sheetsage#30]
I'm sorry if my question is too bold.
Would it be possible to create a sheetsage space in huggingface.co?

**Comment by AgentHitmanFaris** [comment]
Yes, but don't use jukebox as it will not fit for free tier unless you choose to use paid GPU Tier
